modules/cnv-editing-kubevirtstorageclassdefaults-web.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-storage-defaults-for-datavolumes.adoc
+
+[id="cnv-editing-kubevirtstorageclassdefaults-web_{context}"]
+= Editing the `kubevirt-storage-class-defaults` ConfigMap in the web console
+
+You can edit the default storage setting by modifying the `kubevirt-storage-class-defaults` ConfigMap in the `openshift-cnv` namespace. 
+
+If you want to create DataVolumes using different storage settings, you can add storage settings for storage classes in the ConfigMap.
+It is important that you configure storage settings that are supported by the underlying storage.
+
+.Procedure
+
+. Click *Workloads* -> *Config Maps* from the side menu. 
+. In the *Project* list, select *openshift-cnv*.
+. Click *kubevirt-storage-class-defaults* to open the *Config Map Overview*.
```

Comment:
Is it "Config Map" or ConfigMap?

Reply:
In the UI it's 'Config Map'. I'll raise a UI bug